scripts/variantreport_p_sample.rmd

```diff
@@ -34,11 +34,11 @@ outFile <- params$outFile
 ### Sample: `r sample_name` 
 
 ```{r test_files, include = FALSE}
-# location_sigmuts <- '/mnt/beast/pathogenomics/vpipe/akalinlab_pathogenomics/vep_mutation_matching/signature_mutations'
-# sampledir <- '/mnt/beast/pathogenomics/vpipe/akalinlab_pathogenomics/samples/wastewater_all/Test'
+# location_sigmuts <- 'tests/databases/sigmut_db'
+# sampledir <- 'tests'
 # # vepfile <- '/mnt/beast/pathogenomics/pigx_sarscov2_ww/output/vep/Test_vep_sarscov2_reportinput.txt' 
-# vepfile <- '/mnt/beast/pathogenomics/pigx_sarscov2_ww/output/vep/report_ready.txt' 
-# snvfile <- '/mnt/beast/pathogenomics/vpipe/akalinlab_pathogenomics/samples/wastewater_all/Test/variants/SNVs/snvs.csv'
+# vepfile <- 'tests/sample_data/Test_vep_sarscov2_parsed.txt' 
+# snvfile <- 'tests/sample_data/Test_snv.csv'
```

Comment:
Since these lines are commented could they instead be removed?

Reply:
hm, I use them to be able to test directly if this script still works when I'm working on it. 
Would it be ok, if we remove them at a later stage when we have reasonable tests and testing procedures in place?

Reply:
Sure.

I guess we should take some time to make that happen sooner.  How does this report get its "real" arguments now?